README.md

```diff
@@ -8,6 +8,73 @@ so it can directly be executed as a native application on the target platform.
 [![Travis CI](https://api.travis-ci.org/gluonhq/client-maven-plugin.svg?branch=master)](https://travis-ci.org/gluonhq/client-maven-plugin)
 [![BSD-3 license](https://img.shields.io/badge/license-BSD--3-%230778B9.svg)](https://opensource.org/licenses/BSD-3-Clause)
 
+### Requirements
+
+#### Mac OS X and iOS
+
+* Download this version of Graal VM: https://download2.gluonhq.com/substrate/graalvm/graalvm-svm-darwin-20.1.0-ea+28.zip and unpack it like you would any other JDK. (e.g. in `/opt`)
+
+* Configure the runtime environment. Set `GRAALVM_HOME` environment variable to the GraalVM installation directory.
+
+For example:
+
+    export GRAALVM_HOME=/opt/graalvm-svm-darwin-20.1.0-ea+28
+
+* Set `JAVA_HOME` to point to the GraalVM installation directory
+
+For example:
+
+    export JAVA_HOME=$GRAALVM_HOME
+
+##### Additional requirements
+
+* iOS can be built only on Mac OS X
+
+* Install `Homebrew`, if you haven't already. Please refer to https://brew.sh/ for more information.
+
+* Install `libusbmuxd`
+
+Using `brew`:
+
+    brew install --HEAD libusbmuxd
+
+* Install `libimobiledevice`
+
+Using `brew`:
+
+    brew install --HEAD libimobiledevice
+
+#### Linux and Android
+
+* Download this version of Graal VM: https://download2.gluonhq.com/substrate/graalvm/graalvm-svm-linux-20.1.0-ea+28.zip and unpack it like you would any other JDK. (e.g. in `/opt`)
+
+* Configure the runtime environment. Set `GRAALVM_HOME` environment variable to the GraalVM installation directory.
+
+For example:
+
+    export GRAALVM_HOME=/opt/graalvm-svm-linux-20.1.0-ea+28
+
+* Set `JAVA_HOME` to point to the GraalVM installation directory
+
+For example:
+
+    export JAVA_HOME=$GRAALVM_HOME
+
+##### Additional requirements
+
+* Android can be built only on Linux OS
+
+The client plugin will download the Android SDK and install the required packages. 
+
+Alternatively, you can define a custom location to the Android SDK by setting the `ANDROID_SDK` and `ANDROID_NDK` environment variables, making sure that you have installed all the packages from the following list:
+
+* platforms;android-27
+* platform-tools
+* build-tools;27.0.3
+* ndk-bundle
```

Comment:
User can either download the Android NDK and set the env var `ANDROID_SDK` or they can download `ndk-bundle` from `ANDROID_SDK` and there is no need to `ANDROID_SDK` as [substrate falls back to its location](https://github.com/gluonhq/substrate/blob/master/src/main/java/com/gluonhq/substrate/model/InternalProjectConfiguration.java#L222) automatically. This section should be improved to add this information.

Reply:
If you install manually NDK, it might be in a different location, therefore you need to set `ANDROID_NDK` as well.

Reply:
Sure. In that case `ndk-bundle` is not required.

Reply:
Well, NDK is required, whether you download it separately or not. We can point to its download site.